packages/nextjs/app/page.tsx

```diff
@@ -3,8 +3,14 @@
 import Link from "next/link";
 import type { NextPage } from "next";
 import { BugAntIcon, MagnifyingGlassIcon } from "@heroicons/react/24/outline";
+import { useScaffoldReadContract } from "~~/hooks/scaffold-eth";
 
 const Home: NextPage = () => {
+  const { data: totalCounter } = useScaffoldReadContract({
```

Comment:
I'm not really a fan of using ambiguous variables, what if we name it something like `totalCheckedInStudents`?

Reply:
You're right, I updated it to `totalCheckedInStudents` in commit https://github.com/BuidlGuidl/batch9.buidlguidl.com/pull/19/commits/4e28c10bc91659b80e61c1549614b2f9b2719020.